lib/DependenciesBlockVariable.js

```diff
@@ -40,11 +40,10 @@ class DependenciesBlockVariable {
 
 	hasDependencies(filter) {
 		if(filter) {
-			if(this.dependencies.some(filter)) return true;
+			return this.dependencies.some(filter);
 		} else {
-			if(this.dependencies.length > 0) return true;
+			return this.dependencies.length > 0;
```

Comment:
You can omit `else` block and just type this statement on the end of the function